templates/cnx-etcd.yaml

```diff
@@ -43,7 +43,10 @@ data:
   tigera.cnx-manager.error-tracking: "false"
   # Enable ALP support in the UI
   tigera.cnx-manager.alp-support: "false"
-
+  # The name of the cluster.  This field is used as part of the index name of Elasticsearch logs, and is intended
+  # to allow multiple clusters to share one Elasticsearch cluster.  The value of this field must match that of 
+  # ELASTIC_INDEX_SUFFIX in tigera-fluentd-node.
+  tigera.cnx-manager.cluster-name: "cluster"
```

Comment:
Do we want to make this a config option to allow mutliple charms to hook up to a single elastic?

Reply:
Good point. This would be nice to support, but I think it's a very low priority (wishlist item) unless we get feedback from users requesting this.

I've opened a bug to track it: https://bugs.launchpad.net/charm-tigera-secure-ee/+bug/1820896